frontend/util.lua

```diff
@@ -564,4 +564,18 @@ function util.htmlToPlainTextIfHtml(text)
     return text
 end
 
+-- Encode the HTML entities in a string.
```

Comment:
Should be three `---`.

Reply:
Also I think `Encode HTML entities in string` (no period) would fit better with the overall doc-language.

PS I still have to fix up the auto-documentation push since the move to CircleCI. 😊

Reply:
I don't know why I removed the third -, it was there in the Turbo.lua Escape module. :)